src/com/google/javascript/jscomp/CommandLineRunner.java

```diff
@@ -1185,6 +1195,29 @@ protected CompilerOptions createOptions() {
 
     options.setConformanceConfigs(loadConformanceConfigs(flags.conformanceConfigs));
 
+    if ( ! flags.instrumentationFile.isEmpty()) {
+        String instrumentationPb;
+        Instrumentation.Builder builder = Instrumentation.newBuilder();
+        try (BufferedReader br = new BufferedReader(new FileReader(flags.instrumentationFile))) {
+            StringBuilder sb = new StringBuilder();
+            String line = br.readLine();
+
+            while (line != null) {
+                sb.append(line);
+                sb.append(System.lineSeparator());
+                line = br.readLine();
+            }
+            instrumentationPb = sb.toString();
+            TextFormat.merge(instrumentationPb, builder);
+
+            // Setting instrumentation template
+            options.instrumentationTemplate = builder.build();
+
+          } catch (IOException e) {
```

Comment:
The catch block should be indented a couple of spaces in, same column as the try.